docs(graphql): add remoteResponse directive (#120)
dgraph-io/dgraph-docs: content/graphql/custom/directive.md

Just defining the connection is all it takes and then you can ask a single GraphQL query that performs a local query and joins with (potentially many) remote data sources.

### RemoteResponse directive

In combination with the `@remote` directive, in a GraphQL schema you can also use the `@remoteResponse` directive.
You can define the `@remoteResponse` directive on the fields of a `@remote` type in order to map the JSON key response of a custom query to a GraphQL field.

> For example, for the given GraphQL schema:

Reviewer: ```suggestion
For example, in the given GraphQL schema, we have defined a custom DQL query, whose JSON response contains the results of the `groupby` clause in the `@groupby` key. So, we are using the `@remoteResponse` directive to map the `groupby` field in `GroupUserMapQ` type to the `@groupby` key in the JSON response:
```

> For example, in the given GraphQL schema, we have defined a custom DQL query, whose JSON response contains the results of the `groupby` clause in the `@groupby` key. So, we are using the `@remoteResponse` directive to map the `groupby` field in `GroupUserMapQ` type to the `@groupby` key in the JSON response:

Author: ```suggestion
For example, in the given GraphQL schema there's a defined custom DQL query, whose JSON response contains the results of the `groupby` clause in the `@groupby` key. By using the `@remoteResponse` directive you'll map the `groupby` field in `GroupUserMapQ` type to the `@groupby` key in the JSON response:
```
